flake.lock

```diff
@@ -15,25 +15,62 @@
         "type": "github"
       }
     },
+    "flake-utils_2": {
+      "locked": {
+        "lastModified": 1676283394,
+        "narHash": "sha256-XX2f9c3iySLCw54rJ/CZs+ZK6IQy7GXNY4nSOyu2QG4=",
+        "owner": "numtide",
+        "repo": "flake-utils",
+        "rev": "3db36a8b464d0c4532ba1c7dda728f4576d6d073",
+        "type": "github"
+      },
+      "original": {
+        "owner": "numtide",
+        "repo": "flake-utils",
+        "type": "github"
+      }
+    },
```

Comment:
I've updated our flake so that there is only one `flake-utils`.

> There can be only one ⚔️
> [Highlander](https://www.wikiwand.com/en/Highlander_(film))

Reply:
Excellent, thanks a lot! And I think I didn't even realize that this file was generated :innocent: Will watch the Highlander - in general, if anyone has a movie reference, I most probably didn't watch the movie. So I'm learning lots :slightly_smiling_face: